inabox/tests/test_harness.go

```diff
@@ -0,0 +1,98 @@
+package integration_test
+
+import (
+	"context"
+	"math/big"
+	"net"
+
+	"github.com/Layr-Labs/eigenda/api/clients"
+	clientsv2 "github.com/Layr-Labs/eigenda/api/clients/v2"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/payloaddispersal"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/payloadretrieval"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/verification"
+	"github.com/Layr-Labs/eigenda/common"
+	routerbindings "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDACertVerifierRouter"
+	verifierv1bindings "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDACertVerifierV1"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/Layr-Labs/eigenda/inabox/deploy"
+	"github.com/Layr-Labs/eigenda/test/testbed"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/testcontainers/testcontainers-go"
+	"google.golang.org/grpc"
+)
+
+// InfrastructureHarness contains the shared infrastructure components
+// that are global across all tests (external dependencies)
+type InfrastructureHarness struct {
+	// Infrastructure containers - truly global
+	AnvilContainer      *testbed.AnvilContainer
+	GraphNodeContainer  *testbed.GraphNodeContainer
+	LocalstackContainer *testbed.LocalStackContainer
+	ChainDockerNetwork  *testcontainers.DockerNetwork
+
+	// EigenDA components
+	// TODO: Should EigenDA components be their own test harness?
+	ChurnerServer   *grpc.Server
+	ChurnerListener net.Listener
+
+	// Global configuration
+	TemplateName      string
+	TestName          string
+	InMemoryBlobStore bool
+	LocalStackPort    string
+
+	// DynamoDB table names (global for the test suite)
+	MetadataTableName   string
+	BucketTableName     string
+	MetadataTableNameV2 string
+
+	// Deployment configuration (shared)
+	TestConfig *deploy.Config
+
+	// Logger
+	Logger logging.Logger
```

Comment:
don't think we need this here. this should be per test actually. different tests can use different log levels.

Reply:
This logger is for infrastructure deployment related, not used inside the actual tests currently.